Formula/otp-cli.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# brew config for otp-cli
+class OtpCli < Formula
```

Comment:
These shouldn't be here
```suggestion
class OtpCli < Formula
```